packages/evolution-common/src/utils/__tests__/ParamsValidatorUtils.test.ts

```diff
@@ -0,0 +1,606 @@
+        });
+    });
+
+    describe('isUuid', () => {
+        test('should return no errors for a valid uuid', () => {
+            const errors = ParamsValidatorUtils.isUuid('attr', '123e4567-e89b-12d3-a456-426614174000', 'TestClass');
+            expect(errors).toEqual([]);
+        });
+
+        test('should return no errors for an undefined value', () => {
+            const errors = ParamsValidatorUtils.isUuid('attr', undefined, 'TestClass');
+            expect(errors).toEqual([]);
+        });
+
+        test('should return an error for an invalid uuid', () => {
+            const errors = ParamsValidatorUtils.isUuid('attr', 'invalid', 'TestClass');
+            expect(errors).toHaveLength(1);
+            expect(errors[0].message).toContain('should be a valid uuid');
+        });
+    });
+
+    describe('isPositiveInteger', () => {
+        test('should return no errors for a positive integer', () => {
+            const errors = ParamsValidatorUtils.isPositiveInteger('attr', 123, 'TestClass');
+            expect(errors).toEqual([]);
+        });
+
+        test('should return no errors for an undefined value', () => {
+            const errors = ParamsValidatorUtils.isPositiveInteger('attr', undefined, 'TestClass');
+            expect(errors).toEqual([]);
+        });
+
+        test('should return an error for a negative integer', () => {
+            const errors = ParamsValidatorUtils.isPositiveInteger('attr', -123, 'TestClass');
+            expect(errors).toHaveLength(1);
+            expect(errors[0].message).toContain('should be a positive integer');
+        });
+    });
+
+    describe('isPositiveNumber', () => {
+        test('should return no errors for a positive number', () => {
+            const errors = ParamsValidatorUtils.isPositiveNumber('attr', 123.45, 'TestClass');
+            expect(errors).toEqual([]);
+        });
+
+        test('should return no errors for an undefined value', () => {
+            const errors = ParamsValidatorUtils.isPositiveNumber('attr', undefined, 'TestClass');
+            expect(errors).toEqual([]);
+        });
+
+        test('should return an error for a negative number', () => {
+            const errors = ParamsValidatorUtils.isPositiveNumber('attr', -123.45, 'TestClass');
+            expect(errors).toHaveLength(1);
+            expect(errors[0].message).toContain('should be a positive number');
+        });
+    });
+
+    describe('isDateString', () => {
+        test('should return no errors for a date string', () => {
+            const errors = ParamsValidatorUtils.isDateString('attr', '2024-01-01', 'TestClass');
+            expect(errors).toEqual([]);
+        });
+
+        test('should return no errors for an undefined value', () => {
+            const errors = ParamsValidatorUtils.isDateString('attr', undefined, 'TestClass');
+            expect(errors).toEqual([]);
+        });
+
+        test('should return no errors for an undefined date string', () => {
```

Comment:
identique au test précédent. Ces tests pourraient être des tests paramétrés

Reply:
removed duplicate... let's say 3 is ok for non-parameteric tests!